.github/workflows/webpack-jumpstart.yml

```diff
@@ -11,16 +11,21 @@ jobs:
       pull-requests: none
     timeout-minutes: 20
     env:
-      MERGE_SHA: ${{ github.event.pull_request.merge_commit_sha || github.sha }}
+      BASE_SHA: ${{ github.sha }}
+      HEAD_SHA: ${{ github.event.pull_request.head.sha || github.sha }}
     steps:
       - name: Build jumpstart tarball
         run: |
           id="$(docker container create -u node -w /home/node node /bin/sh -c \
-            'git init cockpit &&
+            "set -x &&
+             git config --global user.email '<>' &&
+             git config --global user.name 'nobody' &&
+             git init cockpit &&
              cd cockpit &&
-             git fetch --depth 1 https://github.com/cockpit-project/cockpit "'"${MERGE_SHA}"'":build &&
-             git checkout build &&
-             pkg/build webpack-jumpstart.tar -j$(nproc)'
+             git fetch --no-tags https://github.com/cockpit-project/cockpit ${BASE_SHA}:base ${HEAD_SHA}:head &&
+             git checkout --detach ${BASE_SHA} &&
```

Comment:
Stylistic: You defined the "base" alias for BASE_SHA, so you may just as well use it? (I'd actually be fine with just directly using the macros, and drop the aliases)

Reply:
I wanted to use the sha directly so the merge doesn't go on the "base" branch.  I only added `--detach` later to suppress the warning about it, but in fact, with `--detach`, I can use the branch name and still get a detached head.  I'll do this.

Reply:
done.